Arrow/Sources/Arrow/ProtoUtil.swift

```diff
@@ -64,6 +64,22 @@ func fromProto( // swiftlint:disable:this cyclomatic_complexity function_body_le
             let arrowUnit: ArrowTime64Unit = timeType.unit == .microsecond ? .microseconds : .nanoseconds
             arrowType = ArrowTypeTime64(arrowUnit)
         }
+    case .timestamp:
+        let timestampType = field.type(type: org_apache_arrow_flatbuf_Timestamp.self)!
+        let arrowUnit: ArrowTimestampUnit
+        switch timestampType.unit {
+        case .second:
+            arrowUnit = .seconds
+        case .millisecond:
+            arrowUnit = .milliseconds
+        case .microsecond:
+            arrowUnit = .microseconds
+        case .nanosecond:
+            arrowUnit = .nanoseconds
+        }
+
+        let timezone = timestampType.timezone
+        arrowType = ArrowTypeTimestamp(arrowUnit, timezone: timezone?.isEmpty == true ? nil : timezone)
```

Comment:
Can we omit `== true` here?

```suggestion
        arrowType = ArrowTypeTimestamp(arrowUnit, timezone: timezone?.isEmpty ? nil : timezone)
```

Reply:
Unfortunately, `isEmpty` may be called in a `nil` value, thus it can be something else then `true` or `false`.